### ISSUE TITLE
CodeIgniter 3.x Compatibilty

### ISSUE THREAD

**Issue by fillipefeitosa** [mdisec/codeigniter-object-inj#1]
Hello There ant thanks for the great article on the [link].(https://www.mehmetince.net/codeigniter-object-injection-vulnerability-via-encryption-key/)

Any chances that this exploit is still working on Codeigniter 3.x ?
as the documentation reads:

```
CodeIgniter comes with a few session storage drivers:

* files (default; file-system based)
* database
* redis
* memcached
```

So if a application runs the default config, with files (cookies), and the core still using unseriliazed method, it still possible, right? 

thanks in advance.

**Comment by mdisec** [comment]
Nope, I dont think so. They must have changed something in their code bases. I will check it out once I've a free time.
